web/src/search_suggestion.ts

```diff
@@ -464,6 +466,13 @@ export function get_topic_suggestions_from_candidates({
     return topics;
 }
 
+function ignore_resolved_topic_prefix(topic_name: string): string {
```

Comment:
It's probably worth double checking on CZO (if you haven't already) that we actually want to do this, since there might be some benefit to sorting the resolved topics below unresolved.